### PR TITLE
fix issue with non apng gif stickers being uploaded

### PR DESCRIPTION
**Pull request by sapphonie**
It seems that `getStickerLink` will occasionally give back a `.png` on a gif sticker, despite the `.png` not even being an APNG. I don't know if this is a discord bug or not, I haven't dug that deep, but it seems like this is the simplest solution.

Apologies if this is poor style, I don't write much TS/JS.

Thanks
Sappho

### CODE REVIEW

File: src/plugins/fakeNitro/index.ts [hunk 1]
```diff
@@ -787,7 +787,13 @@ export default definePlugin({
                 if (sticker.available !== false && (canUseStickers || sticker.guild_id === guildId))
                     break stickerBypass;
 
-                const link = this.getStickerLink(sticker.id);
+                // [12/12/2023] 
+                // Work around an annoying bug where getStickerLink will return StickerType.GIF,
+                // but will give us a normal non animated png for no reason
```
Comment (Nuckyz) [inline src/plugins/fakeNitro/index.ts:792]:
```suggestion
                // [12/12/2023] 
                // Work around an annoying bug where getStickerLink will return StickerType.GIF,
                // but will give us a normal non animated png for no reason
                // TODO: Remove this workaround when it's not needed anymore
```

**Review by Nuckyz — CHANGES_REQUESTED** [review]

File: src/plugins/fakeNitro/index.ts (hunk 1 above)
Comment (sapphonie) [reply src/plugins/fakeNitro/index.ts:792]: 👍

**Review by Nuckyz — APPROVED** [review]

**Comment by Nuckyz** [comment]
Lint is failing

**Comment by sapphonie** [comment]
> Lint is failing

Resolved

**Comment by Nuckyz** [comment]
Thank you!

**Pull request merged**